src/css/home.scss

```diff
@@ -208,30 +323,18 @@
     }
 }
 
-.discord {
-    background-color: $background-med-color;
+.mailing-list {
+    background-color: $link-color;
```

Comment:
Haha we should probably come up with a better name at some point.

Reply:
Addressed in c6787a4! Hopefully you like the new name :)